src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/superclone/InputSuperCloneMethodReference.java

```diff
@@ -0,0 +1,21 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.superclone;
+
+import java.util.ArrayList;
+
+/** Config = default */
+public class InputSuperCloneMethodReference extends ArrayList {
+
+    public interface CheckedSupplier<R, E extends Exception> {
+        R get() throws E;
+    }
+
+    public Object clone() {
```

Comment:
please put `// ok` here, after merge of Nick's PR, it will be required to have at least one ok or violation comment.

Reply:
done